### PR TITLE
docs: add OIDC docs and examples for Google, GitLab

### PR DESCRIPTION
**Pull request by unknwon**
Examples for Okta and Keycloak will come after.

### CODE REVIEW

File: content/docs/administration/sso/oauth2.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Bytebase provides templates for configuring built-in OAuth providers.
 
 ### GitLab
 
-1. Follow the [Configure GitLab as an OAuth 2.0 authentication identity provider](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app) to create an OAuth 2 application in GitLab.
+1. Follow the [Configure GitLab as an OAuth 2.0 authentication identity provider](https://docs.gitlab.com/ee/integration/oauth_provider.html) to create an OAuth 2 application in GitLab.
```
Comment (unknwon) [inline content/docs/administration/sso/oauth2.md:72]: FYI @boojack

File: content/docs/_layout.md [hunk 2]
```diff
@@ -181,6 +181,8 @@ expand_section_list: ["Introduction", "Get Started"]
 
 #### [OAuth 2.0](/administration/sso/oauth2)
 
+#### [OpenID Connect](/administration/sso/oidc)
```
Comment (tianzhou) [inline content/docs/_layout.md:184]:
```suggestion
#### [OpenID Connect (OIDC)](/administration/sso/oidc)
```

File: content/docs/administration/sso/oidc.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+Please make sure the [`--external-url`](/docs/get-started/install/external-url) is configured correctly for the Bytebase instance.
+
+If your start Bytebase with `--external-url https://bytebase.example.com`, then your application callback or redirect URL should be `https://bytebase.example.com/oidc/callback`.
+
+</hint-block>
+
+Basic information:
+
+- **Name**: the display name shwon to your users (e.g. "Google" will be shown as "Sign in with Google")
```
Comment (tianzhou) [inline content/docs/administration/sso/oidc.md:19]: ### 
```suggestion
- **Name**: the display name shown to your users (e.g. "Google" will be shown as "Sign in with Google")
```

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/docs/administration/sso/oauth2.md (hunk 1 above)
Comment (boojack) [reply content/docs/administration/sso/oauth2.md:72]: Thanks

**Pull request merged**